installer/openSUSE/prepare

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific continuation of the prepare.sh script.
+
+PKGEXT='rpm'
+DISTROAKA='openSUSE'
+
+
+# install_dist: see install() in prepare.sh for details.
+install_dist() {
+    local pkgs='' pkgsasdeps='' params='' asdeps=''
+    while [ ! "$#" = 0 ]; do
```

Comment:
`!=`...this is another thing we need to fix in the rest of the codebase :)

(here and elsewhere)